assets/css/university-partners.css

```
@@ -0,0 +1,265 @@
+
+.uni-partners__item {
+  display: inline-flex;
+  align-items: center;
+  justify-content: center;
+  height: 5rem;
+  min-width: 8.5rem;
+  padding: 0.7rem 0.95rem;
+  border-radius: 0.6rem;
+ 
+
+  text-decoration: none;
```

Stylelint (17.12.0)

[error] Expected no empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)